### PR TITLE
Form css

### PR DESCRIPTION
**Pull request by gorzat**
form
zrzut ekranu 2017-03-09 o 15 03 49

list
zrzut ekranu 2017-03-09 o 15 13 45

### CODE REVIEW

**Review by matzimowski — APPROVED** [review]

File: 02 - HTML intro/form.html [hunk 1]
```diff
@@ -4,14 +4,21 @@
         <meta charset="UTF-8">
         <title>Form</title>
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.css
+
+" rel=stylesheet>
```
Comment (kdzwinel) [inline 02 - HTML intro/form.html]: you don't need that `\n` after `reset.css`

File: 02 - HTML intro/form.html [hunk 2]
```diff
@@ -4,14 +4,21 @@
...
+        <link href="https://fonts.googleapis.com/css?family=Merriweather:400,900|Open+Sans+Condensed:300" rel="stylesheet">
+
+        <link href="form-style.css" rel=stylesheet>
```
Comment (kdzwinel) [inline 02 - HTML intro/form.html]: rel="stylesheet"

File: 02 - HTML intro/form.html [hunk 3]
```diff
@@ -32,17 +39,28 @@
                     <label for="webpage">Link to your webpage:</label>
                     <input id="webpage" type="url" required>
                 </p>
+                <div class="form-radio">
+                    <p>
+                        Which fruit do you prefere?
```
Comment (kdzwinel) [inline 02 - HTML intro/form.html]: prefer?

File: 03 - CSS/list.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Form</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="list-style.css" rel=stylesheet>
```
Comment (kdzwinel) [inline 03 - CSS/list.html]: rel="stylesheet"

**Comment by kdzwinel** [comment]
Love the colors. I prefer banana. Haven't seen Gravity Falls, all other cartoons 10/10.

**Review by burakukula — APPROVED** [review]

**Review by kdzwinel — CHANGES_REQUESTED** [review]
please see the comments ⬆️

**Review by kdzwinel — APPROVED** [review]
go!

**Pull request merged**